views/amendCaseNote.njk

```diff
@@ -58,7 +58,7 @@
         </div>
     {% endfor %}
 
-    <form action="{{ postAmendment }}" method="post">
+    <form action="{{ postAmendment }}" method="post" autocomplete="off">
```

Comment:
The select option wasn't being set at the time of the `ready` event, but later on so couldn't use the javascript to load the children.  According to https://stackoverflow.com/questions/8861181/clear-all-fields-in-a-form-upon-going-back-with-browser-back-button ended up with this solution